mirdata/datasets/saraga_carnatic.py

```diff
@@ -433,16 +441,17 @@ def load_sections(sections_path):
     intervals = []
     section_labels = []
 
-    with open(sections_path, 'r') as reader:
-        for line in reader.readlines():
+    with open(sections_path, 'r') as fhandle:
+        reader = csv.reader(fhandle, delimiter='\t')
+        for line in reader:
             if line != '\n':
```

Comment:
I think this will always be false, maybe check instead if line[0] is an empty string?

Reply:
Yes. This was useful with the other implementation since some annotation files have an additional \n at the end.